bindings/go/osvschema/vulnerability.go

```diff
@@ -0,0 +1,170 @@
+package osvschema
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// Package identifies the affected code library or command provided by the
+// package.
+//
+// See: https://ossf.github.io/osv-schema/#affectedpackage-field
+type Package struct {
+	Ecosystem Ecosystem `json:"ecosystem"      yaml:"ecosystem"`
```

Comment:
I'm not sure if using the `Ecosystem` type is technically correct here. The constants are defined without the `:<RELEASE>` suffixes that ecosystems use
(e.g. `EcosystemAlpine Ecosystem = "Alpine"`, but this would hold `Alpine:v3.16`)

Reply:
Ecosystem should just be string here I believe. On osv-scanner's side we have the Parsed type, but that's not available in osv-schema (since it requires some opinionated decisions.)

Reply:
okay, I've made it a string